src/pages/[language]/[...slug]/index.query.graphql

```diff
@@ -233,3 +236,38 @@ fragment link on RecordInterface {
     url
   }
 }
+
+fragment linkModel on LinkRecord {
```

Comment:
Could you explain when we use `linkModel` and when to use `linkBlock`? Don't fully understand this addition tbh.

Reply:
Blocks can only be used as part of a modular content field. So if we would want an optional single link to be added to an image card, we would need a modular content field with a maximum of one link block. Would that be your preference?

Reply:
I think it would be my preference, the current 'link situation' is a bit messy and adding more options to add links code-wise feels like adding on to the complexity.

> a modular content field with a maximum of one link block
If I'm not mistaken we do this for one of the landing page sections already.